arbor/backends/gpu/shared_state.hpp

```diff
@@ -118,7 +129,34 @@ struct ARB_ARBOR_API istim_state {
     istim_state() = default;
 };
 
+struct mech_storage {
+    array data_;
+    iarray indices_;
+    std::vector<arb_value_type>  globals_;
+    std::vector<arb_value_type*> parameters_;
+    std::vector<arb_value_type*> state_vars_;
+    std::vector<arb_ion_state>   ion_states_;
+    memory::device_vector<arb_value_type*> parameters_d_;
+    memory::device_vector<arb_value_type*> state_vars_d_;
+    memory::device_vector<arb_ion_state>   ion_states_d_;
+    random_numbers random_numbers_;
+    ARB_SERDES_ENABLE(mech_storage, data_, random_numbers_);
+};
```

Comment:
this is repeated within the struct `shared_state`, please remove either.

Reply:
Yupp, merge played a trick on me!